sweepai/handlers/create_pr.py

```diff
@@ -59,9 +59,12 @@ def create_pr(
     try:
         logger.info("Making PR...")
         pull_request.branch_name = sweep_bot.create_branch(pull_request.branch_name)
-        for fcr in file_change_requests:
-            if not fcr.content.strip():
-                sweep_bot.repo.delete_file(fcr.filename, "Delete empty file", fcr.sha, branch=pull_request.branch_name)
+        try:
```

Comment:
move the try block around
                if not fcr.content.strip():
                    sweep_bot.repo.delete_file(fcr.filename, "Delete empty file", fcr.sha, branch=pull_request.branch_name)